eva_cttv_pipeline/evidence_strings.py

```diff
@@ -149,8 +168,8 @@ def __init__(self, clinvar_record, report, trait, ensembl_gene_id, cellbase_reco
                             clinvar_record.observed_refs_list +
                             clinvar_record.measure_set_refs_list))
 
-        super().__init__(a_dictionary, clinvar_record,
-                         trait.efo_list, ref_list, ensembl_gene_id, report)
+        super().__init__(a_dictionary, clinvar_record, trait.ontology_id, ref_list,
+                         ensembl_gene_id, report, trait.clinvar_name, trait.ontology_label)
```

Comment:
It would be more flexible to use `trait` as an argument. Would there be any particular reason not to use it?